python/lsst/meas/base/catalogCalculation.py

```diff
@@ -50,27 +51,27 @@ def __init__(self, config, name, schema, metadata):
     def getExecutionOrder(cls):
         ''' Sets the relative order of plugins (smaller numbers run first).
 
-        Afterburner plugins must run with BasePlugin.DEFAULT_AFTERBURNER or higher
+        CatalogCalculation plugins must run with BasePlugin.DEFAULT_CATALOGCALCULATION or higher
 
         All plugins must implement this method with an appropriate run level
         '''
         raise NotImplementedError()
 
-    def burn(self, cat, **kwargs):
+    def calculate(self, cat, **kwargs):
         """!
         Process either a single catalog enter or the whole catalog and produce output defined by the plugin
 
         @param[in,out] cat  Either a lsst source catalog or a catalog entery depending on the plug type
                             specified in the classes configuration. Results may be added to new columns,
                             or existing entries altered.
-        @param[in] kwargs   Any additional kwargs that may be passed through the afterburner task.
+        @param[in] kwargs   Any additional kwargs that may be passed through the CatalogCalculationPlugin.
         """
         raise NotImplementedError()
 
 
-class AbContext(object):
+class CCContext(object):
```

Comment:
Is there a reason this is CCContext as opposed to CatalogCalculationContext (other than the fact that AfterburnerContext was also abbreviated)? For example, this would only be one character longer than CatalogCalculationConfig, which is a class name defined in line 101.